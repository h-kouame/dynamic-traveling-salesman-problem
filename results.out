Run Stats for experiment at: 04-24-2018 19:55:35
Run Stats for experiment at: 04-24-2018 19:56:17
10094.0
9994.0
9973.0
10057.0
10012.0
10105.0
10049.0
9968.0
9867.0
9631.0
10274.0
10143.0
10368.0
10072.0
9875.0
9383.0
9945.0
9495.0
9973.0
9295.0
9940.0
9432.0
9922.0
10024.0
9723.0
10044.0
10091.0
10096.0
9911.0
9750.0
10109.0
9732.0
10232.0
10250.0
9827.0
9376.0
9421.0
10034.0
10008.0
9421.0
10064.0
9439.0
9960.0
9845.0
9752.0
10228.0
9832.0
9707.0
9646.0
10145.0
9572.0
9028.0
9689.0
10038.0
9683.0
9733.0
9976.0
9934.0
9859.0
9842.0
9562.0
9993.0
9895.0
9428.0
9912.0
9698.0
9695.0
10087.0
9481.0
10012.0
9491.0
9950.0
9543.0
10143.0
10198.0
10071.0
10121.0
10233.0
10086.0
10086.0
9583.0
9499.0
9579.0
9677.0
9726.0
9374.0
9819.0
9499.0
9586.0
9841.0
9632.0
9835.0
10168.0
10080.0
10214.0
9729.0
10211.0
9372.0
10275.0
9809.0
Run Stats for experiment at: 04-24-2018 20:29:11
10162.0
9969.0
9596.0
10305.0
10210.0
10079.0
9667.0
9599.0
10050.0
9667.0
9751.0
9800.0
9840.0
9858.0
10071.0
9592.0
9591.0
9459.0
10219.0
9188.0
9963.0
9452.0
9702.0
9864.0
9423.0
9624.0
9922.0
9938.0
9854.0
9402.0
9831.0
9411.0
9893.0
10159.0
10281.0
10199.0
9920.0
9929.0
9818.0
9584.0
9912.0
9992.0
10299.0
9738.0
9595.0
10008.0
9680.0
9860.0
10118.0
9534.0
9158.0
10044.0
9927.0
10087.0
10094.0
10003.0
9944.0
9944.0
10015.0
10084.0
10055.0
9900.0
9357.0
9632.0
9579.0
9946.0
9837.0
9575.0
10199.0
10016.0
9934.0
9924.0
10144.0
10122.0
10180.0
9822.0
10126.0
10418.0
9893.0
9775.0
9918.0
9973.0
9823.0
9192.0
9645.0
10070.0
9442.0
9381.0
9732.0
9776.0
9596.0
9767.0
9741.0
9480.0
9902.0
