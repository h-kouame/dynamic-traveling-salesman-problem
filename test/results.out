Run Stats for experiment at: 06-02-2018 22:01:45
3188.0
3117.0
3046.0
2904.0
2975.0
3116.0
3079.0
2863.0
3159.0
3044.0
3162.0
2987.0
3039.0
2891.0
3008.0
3290.0
3121.0
3069.0
3036.0
2893.0
2638.0
3044.0
3012.0
3083.0
3184.0
3072.0
3295.0
3089.0
3032.0
3164.0
3320.0
3279.0
3140.0
3040.0
3198.0
3056.0
3095.0
2966.0
3065.0
3137.0
3048.0
3028.0
2996.0
2904.0
2999.0
2966.0
3107.0
3012.0
3213.0
2913.0
3055.0
2864.0
3244.0
3125.0
2923.0
3280.0
3055.0
3169.0
3113.0
3091.0
2882.0
3068.0
3096.0
3227.0
3099.0
3087.0
3093.0
2914.0
2916.0
3100.0
3108.0
3127.0
2759.0
3006.0
3067.0
3037.0
2945.0
2916.0
3035.0
3127.0
3057.0
3118.0
2999.0
2682.0
3019.0
3056.0
3056.0
3223.0
2949.0
3158.0
3048.0
2935.0
3005.0
3064.0
3046.0
3140.0
3004.0
3000.0
3007.0
2982.0
